lib/msf/core/web_services/json_rpc_app.rb

```diff
@@ -26,13 +26,29 @@ class JsonRpcApp < Sinatra::Base
 
       set :sessions, {key: 'msf-ws.session', expire_after: 300}
       set :session_secret, ENV.fetch('MSF_WS_SESSION_SECRET', SecureRandom.hex(16))
+      set :api_token_file, ENV.fetch('MSF_API_TOKEN_FILE', nil)
```

Comment:
Minor: The data services use environment variables starting with `MSF_WS_`. Consider renaming so that it is more consistent with existing environment variables (see lib/msf/core/web_services/framework_extension.rb).

Reply:
Perhaps something like `MSF_WS_JSON_RPC_API_TOKEN_FILE`